runtime-modules/project-token/src/migrations.rs

```diff
@@ -0,0 +1,78 @@
+use super::*;
+use frame_support::traits::OnRuntimeUpgrade;
+
+// syntactic sugar for logging.
+#[macro_export]
+macro_rules! log {
+	($level:tt, $patter:expr $(, $values:expr)* $(,)?) => {
+		log::$level!(
```

Comment:
I think you are missing the log crate dependency in Cargo.toml and LOG_TARGET definition